### DISCUSSION TITLE
Use task scheduler to install the tool

### DISCUSSION THREAD

**Discussion by lcworld** [mrikirill/SynologyDDNSCloudflareMultidomain#52] [Ideas]
In order to solve the issue "Cloudflare no longer listed as a DDNS provider after DSM", I used Task scheduler to install the script, at boot time.

I forked the repos, you could have a look to my version.
   https://github.com/lcworld/SynologyDDNSCloudflareMultidomain

In the readme file, look for "Install using Task Scheduler"

**Comment by mrikirill** [comment]
Hi,

Yep like the idea 👍  would you like to open a PR with that instruction or I can add it ?

**Reply by lcworld** [reply]
I have modify the installation script to remve its deletion. 
I don't know if you want this update directly in your repos.

If yes, I will create the PR.
